### DISCUSSION TITLE
Unable to get Mongo aggregation pipeline spans in event breakdown

### DISCUSSION THREAD

**Discussion by abhishek17569** [getsentry/sentry-python#1580] [General]
Hello All,
I'm using pymongo and performing multiple aggregation pipelines. 
I want to get the DB process spans to be shown in event breakdown.

Flask Restx
SDK version: 1.9.3
 
Kindly advise

**Comment by sl0thentr0py** [comment]
Hi @abhishek17569 this is actually a feature request, we'll need to build a pymongo integration.
Tracking [here](https://github.com/getsentry/sentry-python/issues/1585), but fair warning, we won't get to it anytime soon. Pull requests are always welcome!

**Reply by abhishek17569** [reply]
Alright Thank you!
Hope it comes out soon

**Comment by sl0thentr0py** [comment]
we now have a pymongo integration so marking this as answered
